src/Engines/nativeEngine.ts

```diff
@@ -2224,6 +2225,22 @@ export class NativeEngine extends Engine {
         // TODO
     }
 
+    /**
+     * Create a canvas
+     * @param width width
+     * @param height height
+     * @return ICanvas interface
+     */
+    public createCanvas(width: number, height: number) : ICanvas {
+        if (_native.NativeCanvas) {
+            const canvas = new _native.NativeCanvas();
+            canvas.width = width;
+            canvas.height = height;
+            return canvas;
+        }
+        throw new Error("Native Canvas plugin not available.");

```

Comment:
nit: i think this is clearer

```suggestion
        if (!_native.NativeCanvas) {
            throw new Error("Native Canvas plugin not available.");
        }

        const canvas = new _native.NativeCanvas();
        canvas.width = width;
        canvas.height = height;
        return canvas;
```